src/generic/UnivPoly.jl

```diff
@@ -876,6 +876,17 @@ function evaluate(a::S, vars::Vector{S}, vals::Vector{V}) where {S <: UnivPoly{T
    return evaluate(a, varidx, vals)
 end
 
+function (a::Union{MPolyRingElem, UniversalPolyRingElem})(;kwargs...)
+   S = parent(a)
+   vars = Array{Int64}(undef, length(kwargs))
+   vals = Array{RingElement}(undef, length(kwargs))
+   for (i, (var, val)) in enumerate(kwargs)
+     vars[i] = findfirst(isequal(var), S.S)
```

Comment:
Perhaps there should be a new method for `var_index` which takes a polynomial ring and a symbol?